kong/plugins/key-auth/handler.lua

```diff
@@ -132,6 +132,7 @@ local function do_authentication(conf)
   -- retrieve our consumer linked to this API key
 
   local cache = kong.cache
+  local core_cache = kong.core_cache
```

Comment:
I don't think we need these two locals anymore, just use `kong.cache` and `kong.core_cache` directly where needed.